clang/lib/AST/ASTImporter.cpp

```diff
@@ -1894,6 +1894,12 @@ ExpectedType clang::ASTNodeImporter::VisitDependentBitIntType(
                                                         *ToNumBitsExprOrErr);
 }
 
+ExpectedType clang::ASTNodeImporter::VisitPredefinedSugarType(
+    const clang::PredefinedSugarType *T) {
+  return Importer.getToContext().getPredefinedSugarType(T->getKind(),
+                                                        T->desugar());
```

Comment:
You have to import the underlying type first, you can't directly use an AST node from a different context.